src/app/view/components/dot-persona-selector-option/dot-persona-selector-option.component.html

```diff
@@ -0,0 +1,13 @@
+<dot-avatar
+    [label]="persona.name"
+    [showDot]="persona.personalized"
+    [size]="24"
+    [url]="persona.photo"
+></dot-avatar>
+<label [ngClass]="{ 'dot-persona-selector-option__personalized': persona.personalized }">{{
+    persona.name
+}}</label>
+<button *ngIf="persona.personalized" (click)="deletePersonalized($event)">
```

Comment:
We have an icon button: https://github.com/dotCMS/core-web/blob/master/src/app/view/components/_common/pattern-library/pattern-library.component.html#L50